api/src/backend/tasks/tasks.py

```diff
@@ -356,17 +380,23 @@ def get_writer(writer_map, name, factory, is_last):
     upload_uri = _upload_to_s3(tenant_id, compressed, scan_id)
 
     if upload_uri:
-        try:
-            rmtree(Path(compressed).parent, ignore_errors=True)
-        except Exception as e:
-            logger.error(f"Error deleting output files: {e}")
+        # TODO: This should be removed, the s3 integration can't work with this block of code, we need to create a new periodic task to delete the output files
+        # In addition, this task shouldn't be responsible for deleting the output files
+        # try:
+        #     rmtree(Path(compressed).parent, ignore_errors=True)
+        # except Exception as e:
+        #     logger.error(f"Error deleting output files: {e}")
```

Comment:
I'm not sure about if we should delete this in the PR. This may cause the container file system to run out of space.

Reply:
What I proposed is that we could add a cronjob in the container to handle this, I believe it’s a better option than adding code to the S3 integration just to work around a problem that shouldn’t exist. We need to create a proper periodic task for this. We can definitely discuss it, but keeping this code would mean introducing significant changes that will have to be removed anyway once the cleanup task is implemented

Reply:
The problem I see here is that we can't introduce that dependency because this will affect both on-prem and cloud deployments.